src/test/resources/__files/payments/200.get_payment_by_id.unknown_response_field.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id":"c7f3f6c5-ae17-4d43-829c-***",
+  "amount_in_minor":101,
+  "currency":"GBP",
+  "user":{
+    "id":"ac459f9f-37df-4cd7-ada5-***",
+    "name":"Andrea Di Lisio",
+    "email":"andrea@truelayer.com"
+  },
+  "this_is_new": "hello",
```

Comment:
random new field

Reply:
Can we add also a comment here to make clear the purpose of this field?